README.md

```diff
@@ -107,17 +107,29 @@ Consider this "dot" directory as your own dedicated space for prototyping. The f
 
 As a concrete example, the unit tests use a temporary `.test` directory as their environment.
 
-## 4. Generate DAG (directed acyclic graph) files
+## 4. Generate DAGs and container images
 
 Run the following command from the project root:
 
 ```bash
 $ python scripts/generate_dag.py \
     --dataset DATASET_DIR \
-    --pipeline PIPELINE_DIR
+    --pipeline PIPELINE_DIR \
+    [--skip-builds] \
+    [--env] dev
 ```
 
-This generates a Python file that represents the DAG for the pipeline (the dot dir also gets a copy). To standardize DAG files, the resulting Python code is based entirely out of the contents in the `pipeline.yaml` config file.
+This generates a Python file that represents the DAG (directed acyclic graph) for the pipeline (the dot dir also gets a copy). To standardize DAG files, the resulting Python code is based entirely out of the contents in the `pipeline.yaml` config file.
+
+Using `KubernetesPodOperator` requires having a container image available for use. The command above allows this architecture to build and push it to [Google Container Registry](https://cloud.google.com/container-registry) on your behalf. Follow the steps below to prepare your container image:
+
+1. Create an `_images` folder under your dataset folder if it doesn't exist.
+
+2. Inside the `_images` folder, create another folder and name it after what the image is expected to do, e.g. `process_shapefiles`, `read_cdf_metadata`.
+
+3. In that subfolder, create a [Dockerfile](https://docs.docker.com/engine/reference/builder/) and any scripts you need to process the data. Use the [COPY command](https://docs.docker.com/engine/reference/builder/#copy) in your `Dockerfile` to include your scripts in the image.
```

Comment:
Will we include a sample Dockerfile and accompanying scripts? I think we. should and we should refer to them here.

Reply:
I think we should too. Done here: e831bbfb997d11a024daf3900c1998874e3e3929

(I based it out of your [blog post](https://cloud.google.com/blog/topics/developers-practitioners/using-cloud-build-keep-airflow-operators-date-your-composer-environment) by the way)

Reply:
> Some questions
> 
> * How will we be expecting users to represent the requirements for the scripts that will be in their images?  (requirements file? Pipfile?) Are we going to be prescriptive, or is that out of scope? Either way, we may want to mention it and show an example of our preferred way, because their Dockerfile will need to copy in those requirements and do the installation path as welll

I was thinking we let them specify the requirements explicitly in the `Dockerfile` during a `pip install` command. So we prevent contributors from creating three files instead of two:

- `Dockerfile`
- `script.py`
- `requirements.txt`

Although.. for people familiar with Dockerfiles, having a `requirements.txt` file sounds like a more common practice. Should we go for the `requirements.txt` approach?

Reply:
Per our discussion in our meeting, we decided `requirements.txt` sounds like the right approach here - we don't want to be overly prescriptive because in theory a user is bringing in their script with their requirements to be packaged up in an image and it's up to them to decide how to go about writing the Dockerfile, etc. 

The changes we'll need to make are the addition of the `requirements.txt` in diagrams showing repo layout, and adding lines in an example `Dockerfile` that `COPY` the file in, install `pip`, and install the requirements.

Reply:
Thanks for listing out the changes as discussed. Kindly check 67b3eb4 and 21d9015.